Foundation/FoundationErrors.swift

```diff
@@ -225,8 +225,12 @@ internal func _NSErrorWithWindowsError(_ windowsError: DWORD, reading: Bool) ->
         default:
             if reading {
                 switch Int32(bitPattern: windowsError) {
-                    case ERROR_FILE_NOT_FOUND: cocoaError = .fileReadNoSuchFile
-                    case ERROR_PATH_NOT_FOUND: cocoaError = .fileReadNoSuchFile
+                    case ERROR_FILE_NOT_FOUND, ERROR_PATH_NOT_FOUND:
+                        // On an empty path, Windows will return FILE/PATH_NOT_FOUND
+                        // rather than invalid path as posix does
+                        cocoaError = paths?.contains("") ?? false
```

Comment:
This might be better as `paths?.isEmpty` as `.contains("")` will always return `false`.

Reply:
`paths` is a `[String]?` so that's checking if any path it contains is empty as opposed to checking if is `"" ` itself.